azuracast/liquidsoap_custom.liq

```diff
@@ -343,26 +327,21 @@ sources = [
     ({5w and 12h-13h58m}, once(playlist_ritka_pentek)),                #Friday
     ({5w and 15h-15h58m}, once(playlist_aa_radio)),
     ({5w and 16h-16h58m}, once(playlist_randomcsunyalanyok)),           
-    ({5w and 17h-17h58m}, once(playlist_dead_hound)),
     ({5w and 18h-18h58m}, once(playlist_graveyard_slot)),
     ({5w and 20h-20h58m}, once(playlist_moneyka)),
-    ({5w21h30m-6w02h}, external_cosmos),
-    ({5w and 12h30m-22h}, playlist_off_air_ambient),                        #ambient: Ritka pentek - Boombap
-    ({6w and 9h-9h10m}, external_cosmos),                  #Test
-    ({6w and 13h30m-14h58m}, once(playlist_kerodzo)),     #Saturday
+    ({5w and 21h-21h58m}, once(playlist_boombap)),       
+    ({5w and 12h30m-21h30m}, playlist_off_air_ambient),                        #ambient: Ritka pentek - Boombap
+    ({6w and 10h-12h}, once(playlist_superstar)),     #Saturday
+    ({6w and 13h30m-14h58m}, once(playlist_kerodzo)),     
     ({6w and 15h-17h58m}, once(playlist_torso)),
     ({6w and 18h-19h58m}, once(playlist_rnr666)),
     ({6w and 20h-21h58m}, once(playlist_mood_sequence)),
     ({6w and 6w23h-7w00h28m}, once(playlist_d23)),
-    ({6w12h-7w00h}, playlist_off_air_ambient),                     #ambient: Lohuma - D23
+    ({6w10h30m-7w00h}, playlist_off_air_ambient),                     #ambient: Superstar - D23
     ({7w and 10h30m-12h28m}, once(playlist_lohuma)),                    #Sunday
-    ({7w and 14h-16h58m}, once(playlist_donald_kacsa_klub)),
-    ({7w and 17h-17h58m}, once(playlist_geigercounterculture)),
-    ({7w and 18h-18h58m}, once(playlist_hetedik_tipusu_talalkozas)),
-    ({7w and 19h-19h58m}, once(playlist_hatterzaj)),
-    ({7w and 20h-20h58m}, once(playlist_csuszka)),
-    ({7w and 21h-21h58m}, once(playlist_sub_burek)),
-    ({7w and 11h-21h30m}, playlist_off_air_ambient),                        #ambient: Lohuma - Sub Burek
+    ({7w and 14h-14h58m}, once(playlist_donald_kacsa_klub)),
+    ({7w and 15h-23h58m}, once(playlist_2023_ujev)),
```

Comment:
I'd do one slot for Űrmajmok from 15-16h
AND
we don't need the entry for the live (which means that we don't have anything here from 16-20h)
AND
I'd add some buffer for `playlist_2023_ujev` say 20-2h, in case we receive more mixes and for a good mood. 
AND
it shouldn't be `once` I guess (I think it would then stop after one file).

Reply:
True-true, I fixed them